zip-XXXX.rst

```diff
@@ -0,0 +1,451 @@
+addresses and a total number of ZEC per block each address is meant to receive
+from the dev fee. The recommendation will be "ratified" by the miners as the
+network upgrade activates.
+
+The role of dev fee recipients
+------------------------------
+
+Dev fee recipients are distinguished from grant recipients in the scope and
+timelines of their work, as well as the specificity of direction. The intent
+is to allow teams to focus on a core competency, while encouraging research and
+adjacent work.
+
+Dev fee recipients are chosen semi-annually by the Foundation based on their
+ability to move Zcash forward. Recipients will typically be development teams,
+though "full stack" teams that can communicate well with the community, expand
+Zcash usage, and widely share their work should be advantaged.
+
+Recipients shall submit quarterly plans to the community for their efforts, as
+well as monthly progress updates.
+
+All work funded by the dev fee will be open source, under licenses compatible
+with the existing Zcash clients.
+
+Though the Foundation shall periodically judge the efficacy of dev fee
+recipients, deciding on and driving roadmaps for Zcash is the role of dev fee
+recipients, in partnership with the community. This role is neatly balanced by
+users running full nodes and miners, either of which can veto a network upgrade.
+
+While dev fee recipients are not required to work exclusively on Zcash,
+considering the nature of their work, recipients must guarantee they aren't
+obliged to the interests of competing projects.
+
+The role of the principal developer
+-----------------------------------
+
+The role of the principal developer is as a "first among equals" amongst the dev
+fee recipients.
+
+The principal developer shall make a number of guarantees.
+
+1. Zcash shall be their exclusive focus, submitting financials periodically to
+   the Foundation as assurance.
+2. They shall maintain a well-run board and employ a qualified CFO.
+3. In addition to the existing open-source requirements, they shall agree to
+   assign any trademarks or patents relevant to Zcash to the Foundation.
+
+In exchange, the principal developer is granted an indefinite dev fee allocation
+and a wide remit to pursue longer-term research relevant to Zcash, as well as
+a voice on the board of the Foundation. The principal developer will not be
+subject to semi-annual review, though it will submit the same plans required of
+other recipients. The principal developer will only be recommended for removal
+by the Foundation in extraordinary circumstances, including reneging on the
+above guarantees or extreme negligence.
+
+Minimum viable Foundation
+-------------------------
+
+To manage the dev fee and fulfill its community and diligence duties, the
+Foundation shall maintain a board of 5 independent members. Rather than the
+structure in the current bylaws, the board will consist of
+
+* 1 seat voted on by ZEC holders directly.
```

Comment:
How? We don't have a such a system in place, and creating one is nontrivial R&D.

Reply:
100%, I've kicked the can here. I also know we'd be open to helping with this

Reply:
In general, I'm looking to the Foundation to provide a plan to execute most of this- that's why I haven't included timelines and left a number of other governance questions unanswered. I think the Foundation can step up.